internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -19,6 +19,11 @@ import (
 	"github.com/envoyproxy/ai-gateway/internal/extproc/router"
 )
 
+const (
+	anthropic = "anthropic"
+	claude    = "claude"
+)
+
```

Comment:
```suggestion
```

Reply:
again, this is not needed. how does this help developers read code? it's literally the same despite the unnecessary constants.

Reply:
remove the constant